src/ORM/Translatable/BaseUniqueIndexNameGenerator.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Knp\DoctrineBehaviors\ORM\Translatable;
+
+ use Doctrine\DBAL\Schema\AbstractAsset;
+
+class BaseUniqueIndexNameGenerator extends  AbstractAsset implements UniqueIndexNameGeneratorInterface {
+
+    public function generate( $columnNames, $prefix='', $maxSize=30 ) {
+
+        return $this->_generateIdentifierName($columnNames, $prefix, $maxSize);
+    }
+
+
+}
```

Comment:
You should add a newline at the end of the file and fix code styling (spaces, new lines, etc.)